### PR TITLE
Added description and auto_create_subnetworks variables.

### LINKED ISSUE #56: Terraform Import of default network requires recreation on next apply
After importing the _default_ network, the next `terraform apply` requires destruction then recreation due to the following variables changing.
```bash
resource "google_compute_network" "network" {
      ~ auto_create_subnetworks         = true -> false # forces replacement
       ...
      - description                     = "Default network for the project" -> null # forces replacement
      ...
```

I would like to add these to the variables.tf so that they can be set in a user module.

### PR DESCRIPTION
**Pull request by kevensen**
Signed-off-by: Ken Evensen <kdevensen@google.com>

Fixes #56

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -54,3 +54,14 @@ variable "delete_default_internet_gateway_routes" {
...
   default     = "false"
 }
+
+
+variable "description" {
+  description = "An optional description of this resource. The resource must be recreated to modify this field."
+  default     = ""
+}
+
+variable "auto_create_subnetworks" {
```
Comment (morgante) [inline variables.tf]: This should be a boolean I believe.

File: variables.tf [hunk 2]
```diff
@@ -54,3 +54,14 @@ variable "delete_default_internet_gateway_routes" {
   description = "If set, ensure that all routes within the network specified whose names begin with 'default-route' and with a next hop of 'default-internet-gateway' are deleted"
   default     = "false"
 }
+
+
+variable "description" {
```
Comment (morgante) [inline variables.tf]: Let's explicitly define the type.

File: variables.tf (hunk 1 above)
Comment (kevensen) [reply variables.tf]: That was my immediate thought.  In main.tf, it is a string
https://github.com/terraform-google-modules/terraform-google-network/blob/4e7cd17b7c7e466fdec245b05d496cd80750a39a/main.tf#L20-L25

File: variables.tf (hunk 2 above)
Comment (kevensen) [reply variables.tf]: will do

File: variables.tf (hunk 1 above)
Comment (morgante) [reply variables.tf]: Historically, strings were auto-converted to booleans but now we can make explicit booleans.

**Comment by kevensen** [comment]
@aaron-lane Done!

**Pull request merged**